<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemWindow</class>
 <widget class="QDialog" name="NewItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый товар</string>
  </property>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="priceSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> руб.</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="selectImgButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать изображение товара</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addItemButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить товар</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameDescLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeDescLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceDescLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgPreviewLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Превью изображения:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="amountSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> шт.</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="amountDescLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество на складе</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
